extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -438,7 +438,7 @@ export class AmpAnalytics extends AMP.BaseElement {
     request = this.expandTemplate_(request, trigger, event);
 
     // For consistency with amp-pixel we also expand any url replacements.
-    return urlReplacementsForDoc(this.win.document).expandAsync(request)
+    return urlReplacementsForDoc(this.getAmpDoc()).expandAsync(request)
```

Comment:
This has to be the nearest context node you can find. Here, it could be either:

1. `this.element`
2. `event.target`

I think the first option is better, but you need to check where events come from. This is something you should also document in comments very well.